src/Relation/Nullary/Construct/Add/Point.agda

```diff
@@ -8,8 +8,16 @@
 
 module Relation.Nullary.Construct.Add.Point where
 
+open import Algebra.Bundles
```

Comment:
Can we get rid of these now unnecessary imports?

Reply:
whoops! some sort of git error in not reverting this file